blivet/formats/fs.py

```diff
@@ -251,6 +253,14 @@ def label_format_ok(self, label):
     label = property(lambda s: s._get_label(), lambda s, l: s._set_label(l),
                      doc="this filesystem's label")
 
+    def can_assign_uuid(self):
```

Comment:
What about `can_set_uuid` and `can_modify_uuid`? (or `change` or `edit` for that matter).